src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -468,11 +519,6 @@ public FileModel ingestFileSuccess(String datasetId, FileLoadModel fileLoadModel
                 (StringUtils.equals(checksum.getType(), "crc32c") ||
                     StringUtils.equals(checksum.getType(), "md5")));
         }
-
-        logger.info("addFile datasetId:{} objectId:{}", datasetId, fileModel.getFileId());
-        addFile(datasetId, fileModel.getFileId());
```

Comment:
Do you want to keep this for the success case of retryAcquireLockIngestFileSuccess or it's not needed?

Reply:
It looks like this chunk of code still lives a few lines up [Line 466] and checkSuccessfulFileLoad is a newly shared method between etryAcquireLockIngestFileSuccess and ingestFileSuccess. So, I think it's okay.

Reply:
Okay, so the addFile won't be called in retryAcquireLockIngestFileSuccess, only in ingestFileSuccess -- right?

Reply:
Oh, you're totally right. I was thinking that was extraneous since all of my checks were performed before that, but the "addFile" is used for cleanup. Adding that now - thank you!